skytap/resource_skytap_vm_test.go

```diff
@@ -498,10 +391,11 @@ func TestAccCasandra(t *testing.T) {
 		CheckDestroy: testAccCheckSkytapEnvironmentDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config:             testAccSkytapVMConfig_cassandra(newEnvTemplateID, os.Getenv("SKYTAP_TEMPLATE_ID"), os.Getenv("SKYTAP_VM_ID"), uniqueSuffixEnv, 22, "", ""),
+				Config:             testAccSkytapVMConfig_cassandra(newEnvTemplateID, templateID, vmID, uniqueSuffixEnv, 22, "", ""),
```

Comment:
should be name Cassandra?

Reply:
The name Cassandra is not meaningful in this circumstance. I have renamed the test.